app/views/tasks/new.html.erb

```diff
@@ -1,13 +1,11 @@
 <h1>Add a New Task</h1>
-<!-- Step 25: Set the action URL to something appropriate!  In this case, that would be "/tasks" -->
-<form method="POST" action="youre_going_to_have_to_set_this_to_something_appropriate" enctype="multipart/form-data">
-  <!-- <input type="hidden" name="authenticity_token" value="#{env['rack.session'][:csrf]}"> -->
-  <!--
-    Step 22: Create an HTML input with a matching label, per the test failure.
-             The name of the field should be "description".
-
-    Hint: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form has a nice
-    example of a simple HTML form!
-  -->
-  <!-- Step 23: Create a submit button, per the next test failure -->
+<form method="POST" action="/tasks" enctype="multipart/form-data">
+  <input type="hidden" name="authenticity_token" value="#{env['rack.session'][:csrf]}">
+  <div class="task-add">
+    <label for="description">Task Description</label>
+    <input type="text" name="description" id="description" required />
```

Comment:
Nothing in the tests thus far (parts 1 and 2) have dictated that you need the `required` attribute here.